tests/queries/0_stateless/02377_extend_protocol_with_query_parameters.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC2154
+
+unset CLICKHOUSE_LOG_COMMENT
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+$CLICKHOUSE_CLIENT \
```

Comment:
Do we have tests for setting unknown setting? (to be sure that we still accept only setting name or `param_*`)

Reply:
if name doesn't start with `param_` we will treat this "smth" as setting. and settings do check, that nothing that cannot be recognised was set. it is covered by old tests, I just rely on them.